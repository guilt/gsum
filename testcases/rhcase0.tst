Name: rhcase0
Description: Test 10M file filled with 0xff, hash 5%-93% range
File: test.txt 10M fill 0xff
AlgoMatrix: default
Range: 5%-93%
