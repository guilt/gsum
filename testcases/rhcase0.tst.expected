Hash-default-5%-93%: c38572bd18913fd77976ca525325e53c9efd76322f1f76929bf6874101a1c2b4
